Podverse/ClipsTableViewController.swift

```diff
@@ -0,0 +1,151 @@
+//
+//  ClipsTableViewController.swift
+//  Podverse
+//
+//  Created by Mitchell Downey on 6/6/17.
+//  Copyright © 2017 Podverse LLC. All rights reserved.
+//
+
+import UIKit
+
+class ClipsTableViewController: UIViewController {
+
+    var clipsArray = [MediaRef]()
+    
+    @IBOutlet weak var tableView: UITableView!
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        
+        MediaRef.retrieveMediaRefsFromServer() { (mediaRefs) -> Void in
+            self.reloadClipData(mediaRefs: mediaRefs)
+        }
+
+    }
+    
+    func reloadClipData(mediaRefs: [MediaRef]? = nil) {
+        if let mediaRefs = mediaRefs {
+            for mediaRef in mediaRefs {
```

Comment:
here is a more swifty way to do an unwrap and for-in loop you might want to try:
`for mediaRef in mediaRefs ?? [] `
this way, less boilerplate and you don't need the if let because you are actually using an empty array (which will obviously not iterate through since the count is 0)